FE/src/App.tsx

```diff
@@ -1,19 +1,31 @@
-import Box from '@mui/system/Box';
-import Grid from '@mui/system/Unstable_Grid';
+import { useState } from 'react';
 import Stack from '@mui/material/Stack';
 import Dashboard from './components/Dashboard';
 import Header from './components/Header';
+import image from './styles/images/corina-rainer-P2wLo_PzHjU-unsplash.jpg';
 
 const App = () => {
+  const [searchedMovie, setSearchedMovie] = useState('');
```

Comment:
It is okay for now, but later we might need to put this into the redux store

Reply:
let's get it over with :D